frontend/src/app.jsx

```diff
@@ -34,18 +32,6 @@ export default function App() {
       setTasks([...tasks, newTask]);
     }};
 
-    //delete
-    const deleteTask = async (id) => {
-      await fetch(`http://localhost:3001/tasks/${id}`, {
-        method: "DELETE",
-      });
-  
-      setTasks(tasks.filter((task) => task.id !== id));
-      setUpdate(!update);
-    };
-  
-  
-
```

Comment:
This method was actually unused, because you are using the one defined inside `<Task />` component, so I removed it from here to clean up the code.